### PR TITLE
Adding summary to catalog.md

### PR DESCRIPTION
**Pull request by edcdavid**
Adding a summary section in the catalog.md to list test cases counts. Also improved the intro a bit and removed static catalog starter files.

### CODE REVIEW

File: CATALOG.md [hunk 1]
```diff
@@ -1,11 +1,57 @@
...
+
+|Mandatory|Optional|
+|---|---|
+|23|5|
 
-## Test Case Catalog
+## Test Case list
 
-Test Cases are the specifications used to perform a meaningful test. Test cases may run once, or several times against several targets. CNF Certification includes a number of normative and informative tests to ensure CNFs follow best practices. Here is the list of available Test Cases:
+Test Cases are the specifications used to perform a meaningful test.Test cases may run once, or several times against several targets. CNF Certification includesa number of normative and informative tests to ensure CNFs follow best practices.Here is the list of available Test Cases:
```
Comment (rdavid) [inline CATALOG.md:54]: Add space after periods (2 cases).

File: cmd/tnf/generate/catalog/catalog.go [hunk 2]
```diff
@@ -175,17 +164,45 @@ func outputTestCases() { //nolint:funlen
 	sort.Strings(suites)
 
 	// Iterating the map by test and suite names
+	outString = "## Test Case list\n\n" +
+		"Test Cases are the specifications used to perform a meaningful test." +
```
Comment (rdavid) [inline cmd/tnf/generate/catalog/catalog.go:168]: Space is needed after the period.

File: cmd/tnf/generate/catalog/catalog.go [hunk 3]
```diff
@@ -175,17 +164,45 @@ func outputTestCases() { //nolint:funlen
...
+		"Test cases may run once, or several times against several targets. CNF Certification includes" +
+		"a number of normative and informative tests to ensure CNFs follow best practices." +
```
Comment (rdavid) [inline cmd/tnf/generate/catalog/catalog.go:170]: Space is needed after the period.

File: cmd/tnf/generate/catalog/catalog.go [hunk 4]
```diff
@@ -224,27 +281,29 @@ func generateJS(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+func outputIntro() (out string) {
+	return "<!-- markdownlint-disable line-length no-bare-urls -->\n\n" +
```
Comment (rdavid) [inline cmd/tnf/generate/catalog/catalog.go:285]: There is no need for the second new line.

**Review by rdavid — APPROVED** [review]

**Review by greyerof — APPROVED** [review]

**Comment by rdavid** [comment]
Please, regenerate CATALAOG.md after the last fix.

File: cmd/tnf/generate/catalog/catalog.go (hunk 3 above)
Comment (edcdavid) [reply cmd/tnf/generate/catalog/catalog.go]: fixed

**Pull request merged**